README.md

```diff
@@ -5,8 +5,8 @@ Python container images
 Images available on Quay are:
 * CentOS 7 [python-38](https://quay.io/repository/centos7/python-38-centos7)
 * RHEL 8 [python-39-minimal](https://quay.io/repository/sclorg/python-39-minimal-el8)
-* CentOS Stream 9 [python-39-c9s](https://quay.io/repository/sclorg/python-39-c9s)
```

Comment:
It seems that we have that Dockerfile in 3.9 so we can either remove it there as well (from multispec) or add it to build and push action. What do you think?

Reply:
The biggest advantage I see now here is that we have 3.9-minimal based on C9S so the full image on the same platform can be used as a build base image which is something we have documented and supported.
I'd keep it. Could you please add it to the build and push GH action and keep it in the readme?

Reply:
Yes, we opted to not build c9s images when there is a UBI version available, since they are used for the same use case. There is no reason to stop building it other than that (anyone in the community can build their own anyway), so if you are ok with maintaining it on top of the UBI image, then I am ok with that.